### ISSUE TITLE
Add Deprecation warning

### ISSUE THREAD

**Issue by josh-burton** [pinkfish/flutter_places_dialog#14]
Since the Google Places Place Picker is now deprecated you should add a clear warning to this plugin so people are aware before using it.

**Comment by pinkfish** [comment]
Interesting.  What is it replaced with?   Just the places API?

**Comment by josh-burton** [comment]
From [here](https://developers.google.com/places/android-sdk/placepicker)

```
Notice: The Place Picker (Android, iOS) is deprecated as of January 29, 2019. This feature will be turned off on July 29, 2019, and will no longer be available after that date. To continue using the Place Picker with Places SDK for iOS v.2.7.0 through the deprecation period, do NOT disable the Places SDK for iOS. Read the migration guides (Android, iOS) to learn more.
```

Basically the places sdk from play services is now deprecated, and there is a new one: `com.google.android.libraries.places:1.1.0`.

This new SDK doesn't include the place picker, but it does include some form of autocomplete UI.

**Comment by pinkfish** [comment]
Looks like the auto-complete api might be hard to do in flutter because it
renders in a fragment.  There is a compatibility library we can use by the
looks.

Probably should make a flutter places picker that just uses flutter
components and the places api directly.

**Comment by josh-burton** [comment]
Yeah there is a package that does that and works quite well: https://pub.dartlang.org/packages/flutter_google_places

**Comment by pinkfish** [comment]
Nice!  Ok, that didn't exist when I did this package :)

**Comment by pmrajani** [comment]
@pinkfish You should add clear comment for this plugins as now this will not work because classes you used is deprecated now. I have spent so much time to check my code issue and i finally realize this plugins will no more work in android.

**Comment by josh-burton** [comment]
Or perhaps archive this repository so it's clear that way.

**Comment by pmrajani** [comment]
@athornz is their any alternative way to implement place autocomplete view?

**Comment by josh-burton** [comment]
Have you tried the package I mentioned above?
